mino/minogrpc/controller/mod_test.go

```diff
@@ -141,6 +141,7 @@ func TestMiniController_FailGenerateKey_OnStart(t *testing.T) {
 }
 
 func TestMiniController_FailMarshalKey_OnStart(t *testing.T) {
+	t.Skip("Testing if this is the culprit")
```

Comment:
The test failed because there was a trailing `key.cert` file in there. No idea how it got here, I had it, too! Also not sure how to avoid that somebody spends 30 minutes like I did searching why the test fails, only to find the `key.cert` file...